VocaDbWeb/Scripts/Repositories/UserRepository.ts

```diff
@@ -123,14 +123,19 @@ module vdb.repositories {
 
 		public getFollowedArtistsList = (
 			userId: number,
-			paging: dc.PagingProperties, lang: string,
+            paging: dc.PagingProperties, lang: string,

```

Comment:
Hmm, spacing seems to have changed here. It should be specified in .editorconfig already.